sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -60,14 +60,19 @@ void PeriodicExportingMetricReader::DoBackgroundWork()
     auto end            = std::chrono::steady_clock::now();
     auto export_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(end - start);
     auto remaining_wait_interval_ms = export_interval_millis_ - export_time_ms;
-    cv_.wait_for(lk, remaining_wait_interval_ms);
+    cv_.wait_for(lk, remaining_wait_interval_ms, [this]() {
+      if (is_force_wakeup_background_worker_.load(std::memory_order_acquire))
+      {
+        is_force_wakeup_background_worker_.store(false, std::memory_order_release);
```

Comment:
Can store fail then false should be returned?

Reply:
I don't know if it can ever fail. It's atomic assignment operation so should eventually be successful.  Also, this method returns `void`, not sure how to get any failure status from this method.
Let me know if I am missing something :)